content/english/csharp-basics/classes-and-objects.md

```diff
@@ -42,10 +42,6 @@ Let's learn about the different parts in a class below:
 ### Class
 A class is a blueprint or prototype of a new type of object. In general, a class contains three important parts: 
 
-* data fields/instance variables 
-* constructor 
-* methods
-
 <img src="../images/class.png" height="400" alt="A Class blueprint for an object contains instance varialbes/data fields which are data/attributes in the object, constructor which are methods that creates the obejct of the class, and methods, which are behaviors possible for the object."/> 
```

Comment:
does the image show up? I believe there may be a extra "." in the path which causes a broken image